web/web/models/home_hub.py

```diff
@@ -21,32 +21,39 @@ class HomeHub(Model):
                          primary_key=True,
                          nullable=False)
 
-    service_location_id = Column(
-        db.Integer,
-        db.ForeignKey('service_locations.service_location_id'),
-        unique=True,
-        nullable=False)
+    service_location_id = Column(db.Integer,
+                                 db.ForeignKey('service_locations.service_location_id'),
+                                 unique=True,
+                                 nullable=False)
+
     market_id = Column(db.Integer,
                        db.ForeignKey('markets.market_id'),
                        nullable=False)
 
-    is_active = Column(db.Boolean(), default=False, nullable=False)
+    is_active = Column(db.Boolean(),
+                       default=False, 
+                       nullable=False)
 
-    is_archived = Column(db.Boolean(), default=False, nullable=False)
+    is_archived = Column(db.Boolean(), 
+                         default=False, 
+                         nullable=False)
 
-    created_at = Column(TIMESTAMP, default=datetime.utcnow, nullable=False)
+    updated_at = Column(TIMESTAMP, 
+                        nullable=False,
+                        server_default=text('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP'))
```

Comment:
are you reformatting these by hand or is it an automated tool?

Reply:
I went quickly and reformatted by hand because it bothers me . Let me know again which tool you prefer to use and I'll do that from now on! Thanks!

Reply:
Yeah, my bad for not being clear. Here's what is being used: https://github.com/slacgismo/TESS/blob/master/web/requirements.txt#L12